F21/joelanzi/Output Result.tsv

```diff
@@ -0,0 +1,19 @@
+word	lang	expected_lower_case	function_output	correct
+HELLO	en	hello	hello	True
+WORLD	en-US	world	world	True
+cAmEl	en-IE	camel	camel	True
+---OK	en-Latn	---ok	---ok	True
+tAcht	ga	t-acht	t-acht	True
+tACHT	ga	t-acht	t-acht	True
+TACHT	ga	tacht	tacht	True
+nAthair	ga	n-athair	n-athair	True
+nATHAIR	ga	n-athair	n-athair	True
+NATHAIR	ga	nathair	nathair	True
+nÓg	ga-IE	n-óg	n-óg	True
+nÕg	ga-IE	nõg	nõg	True
+nÕg	ga-IE	nõg	nõg	True
+KASIM	tr	kasım	kasım	True
+KASIM	en	kasim	kasim	True
+ΠΌΛΗΣ	el	πόλης	πόλης	True
+官话	zh-Hans	官话	官话	True
+ภาษาไทย	th	ภาษาไทย	ภาษาไทย	True
```

Comment:
We should add more cases to this file in order to achieve wider coverage.

Reply:
We should add more languages to the file, adding sentences instead of only words and punctuations.